build.sbt

```diff
@@ -3718,6 +3718,7 @@ lazy val `engine-runner` = project
               "com.amazonaws",
               "com.google",
               "io.grpc",
+              "io.netty.util.concurrent.AbstractScheduledEventExecutor",
```

Comment:
- Any idea why this fix works?
- Does specifying explicitly the FQN takes precedence any other options?
- And `io.netty.util.concurrent.AbstractScheduledEventExecutor` is really initialized in runtime?

Reply:
I believe that the options that are specified on the cmd line take precedence over any other options. And these are specified on the cmd line. So this `io.netty.util.concurrent.AbstractScheduledEventExecutor` class should be initialized at runtime. I have not checked the report though.